build-system/babel-config/minified-config.js

```diff
@@ -74,10 +74,31 @@ function getMinifiedConfig() {
     sourceMaps: 'inline',
     presets: [presetEnv],
     retainLines: true,
+    // Babel assumptions enables transformation optimizations.
+    // Full details at https://babeljs.io/docs/en/assumptions.
     assumptions: {
+      arrayLikeIsIterable: true,
+      constantReexports: true,
```

Comment:
I don't believe this is always safe, but it should be unnecessary regardless. ESbuild should handle this when it bundles, and it'll have the available information to optimize.